### PR TITLE
[bitnami/mediawiki] Fix Mediawiki host when using ingress

### LINKED ISSUE #8182: Using an ingress with mediawiki
**Which chart**:
MediaWiki 12.4.1

**Describe the bug**
Trying to use an ingress (ingress-nginx) with a ClusterIP service sets the APP_HOST to 127.0.0.1

**To Reproduce**
See above.

**Expected behavior**
To work. With an ingress, the APP_HOST is already known.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.7.1", GitCommit:"1d11fcb5d3f3bf00dbe6fe31b8412839a96b3dc4", GitTreeState:"clean", GoVersion:"go1.17.2"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2", GitCommit:"092fbfbf53427de67cac1e9fa54aaa09a28371d7", GitTreeState:"clean", BuildDate:"2021-06-16T12:59:11Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.5", GitCommit:"aea7bbadd2fc0cd689de94a54e5b7b758869d691", GitTreeState:"clean", BuildDate:"2021-09-15T21:04:16Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

Modifies the helper `mediawiki.host` adding support for ingress, similar to how it is done for other assets.

**Possible drawbacks**

None known.

**Applicable issues**

  - fixes #8182

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/mediawiki/templates/_helpers.tpl [hunk 1]
```diff
@@ -15,11 +15,17 @@ Note, returns 127.0.0.1 if using ClusterIP.
...
 */}}
 {{- define "mediawiki.host" -}}
+{{- if .Values.ingress.enabled }}
+{{- $host := .Values.ingress.hostname | default "" -}}
+{{- default (include "mediawiki.serviceIP" .) $host -}}
+{{- else -}}
 {{- $host := index .Values (printf "%sHost" .Chart.Name) | default "" -}}
 {{- default (include "mediawiki.serviceIP" .) $host -}}
 {{- end -}}
+{{- end -}}
```
Comment (miguelaeh) [inline bitnami/mediawiki/templates/_helpers.tpl:28]:
```suggestion
{{- $host := index .Values (printf "%sHost" .Chart.Name) | default "" -}}
{{- if .Values.ingress.enabled }}
{{- $host := .Values.ingress.hostname | default "" -}}
{{- end -}}
{{- default (include "mediawiki.serviceIP" .) $host -}}
{{- end -}}
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thanks for the PR!
Please take a look to the suggestion

**Comment by migruiz4** [comment]
Suggestion applied

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**